rootfs/api/tests/test_auth.py

```diff
@@ -248,7 +248,7 @@ def test_passwd(self):
         }
         response = self.client.post(url, submit,
                                     HTTP_AUTHORIZATION='token {}'.format(token))
-        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.status_code, 401)
```

Comment:
if this was any time other than beta, I'd say this has to be changed, wrong status code or not. Because we're pre-prod though this is fine.